Sdtrig.tex

```diff
@@ -250,7 +250,8 @@ \section{Trigger Registers}
 
 \begin{steps}{As a result, a debugger can write any supported trigger as
 follows:}
-\item Write 0 to \RcsrTdataOne.
+\item Write 0 to \RcsrTdataOne. (This will result in \RcsrTdataOne containing a
+    non-zero value, since the register is WARL.)
```

Comment:
Actually, should this be `\warl`?

Reply:
Yes. That macro is defined so we should use it, even though it's not used anywhere else in the document yet.